src/Middleware/Spa/SpaServices.Extensions/src/Proxying/SpaProxy.cs

```diff
@@ -158,6 +161,10 @@ private static async Task CopyProxyHttpResponse(HttpContext context, HttpRespons
             context.Response.StatusCode = (int)responseMessage.StatusCode;
             foreach (var header in responseMessage.Headers)
             {
+				if (context.Request.Protocol != "HTTP/1.1" && Http2NotForwardedResponseHeaders.Contains(header.Key, StringComparer.OrdinalIgnoreCase))
```

Comment:
This needs to handle more than 1.1. There are some helpers here:
https://github.com/dotnet/aspnetcore/blob/main/src/Http/Http.Abstractions/src/HttpProtocol.cs

Reply:
Thanks, I completely missed that!